changelogs/fragments/109-add-timeout-parameter-to-api.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- "api* modules - added timeout parameter (https://github.com//pull/109)."
```

Now the URL is broken :-)
```suggestion
- "api* modules - added ``timeout`` parameter (https://github.com/ansible-collections/community.routeros/pull/109)."
```
(and while we're at it, how about formatting `timeout` as tt?)

---

(The broken URL seems to be a result of a bug in GitHub's suggestions UI...)

---

Really strange. Fixed.